deploy-manage/monitor/autoops/cc-autoops-as-cloud-connected.md

```diff
@@ -0,0 +1,22 @@
+---
+applies_to:
+  deployment:
+    self:
+navigation_title: AutoOps for self-managed clusters
+---
+
+# AutoOps for self-managed clusters
+
+Use [AutoOps](/deploy-manage/monitor/autoops.md) with your self-managed cluster through [Cloud Connect](/deploy-manage/cloud-connect.md). 
+
+Cloud Connect enables users of self-managed clusters to use {{ecloud}} services by connecting to the {{ecloud}}. This means you can take advantage of the simplified cluster monitoring, real-time issue detection, and performance recommendations of AutoOps without having to run and manage the underlying infrastructure.
```

Comment:
I have never seen elastic cloud referred to as "the elastic cloud" - sentence is also a little repetitive so it might be good to rephrase a bit